modules/atomic-widgets/assets/js/editor/div-block-view.js

```diff
@@ -171,6 +190,25 @@ const DivBlockView = BaseElementView.extend( {
 				// User is dragging an element from the panel.
 				this.onDrop( event, { at: newIndex } );
 			},
+			onDragging: ( side, event ) => {
+				const currentTargetHeight = event.currentTarget.getBoundingClientRect().height;
+				const $placeholder = this.$el.find( '.elementor-sortable-placeholder' );
+
+				if ( 'horizontal' === this.getDroppableAxis() ) {
```

Comment:
Maybe reduce nesting?

```suggestion
				if ( 'horizontal' === this.getDroppableAxis() && $placeholder.length ) {
```